### ISSUE TITLE
transformTags attribs not being applied?

### ISSUE THREAD

**Issue by mruczek** [apostrophecms/sanitize-html#256]
Hello. I tried using the example transformTags code:

```
const sanitizeHtml = require("sanitize-html");
var dirty = "<ol></ol>";

clean = sanitizeHtml(dirty, {
  transformTags: {
    'ol': function(tagName, attribs) {
        return {
            tagName: 'ul',
            attribs: {
                class: 'foo'
            }
        };
    }
  }
});
```

And I get this output: `'<ul></ul>'`

Isn't it supposed to be `'<ul class="foo"></ul>'` ? I'm sorry if I misunderstood anything, I'm new to node.js. I'm using sanitize-html 1.19.1 and node v8.12.0. Any information you can provide me is greatly appreciated.

**Comment by SuchMonkey** [comment]
Hi.
I ran into the same issue and it seems like this is a bug.

I got it to work when setting the 'allowedClasses' attribute:

```const sanitizeHtml = require("sanitize-html");
var dirty = "<ol></ol>";

clean = sanitizeHtml(dirty, {
  allowedClasses: {
    'ul': ['foo']
  },
  transformTags: {
    'ol': function(tagName, attribs) {
        return {
            tagName: 'ul',
            attribs: {
                class: 'foo'
            }
        };
    }
  }
});
```

Then I tried to generalize this with the wildcard thingy like so:
```
{
  allowedClasses: {
    '*': ['foo']
}
```

This does in fact workout but I noticed that suddenly all classes I applied in my transforms where present in the sanitized version, even tough I did not specify them in the classes array.

In the end this workaround works out just fine for all classes applied in your transform functions (Needless to say that this kind of destroys the class filter feature ;)
```
{
  allowedClasses: {
    '*': []
}
```
And  since I am already here: Applying a role attribute to your elements does not work either. 
Probably because it is not even implemented. It would be a nice to have tho :)

We are using sanitize-html version 1.18.2

Still your issue is not closed yet I figured this bug is still present. I have not yet had a chance to test it with the latest version.
